starter_code/public/stylesheets/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+  padding: 50px;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif; }
+
+a {
+  color: #00B7FF; }
+
+.index {
```

Comment:
your css here is a little weird. Your image doesn't take up the full space. 

you've set the width of the container to 900px but you want it to take up the full size of the screen. When you want to take up the full width of the screen, it's better to set it to width 100% because then it will look the same on all screen sizes. 

```
  background-image: url(../images/cinema.jpg);
  background-size: cover;
  height: 100vh;
  color: white;
  display: flex;
  justify-content: flex-end;
  align-items: center;
  padding-right: 100px;
```